ush/soca/marine_recenter.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+
+import os
+from logging import getLogger
+from typing import Dict, List, Any
+from wxflow import logit
+
+logger = getLogger(__name__.split('.')[-1])
+
+
+class MarineRecenter():
+
+    def __init__(self, config):
```

Comment:
Please add stub documentation and follow `typing` hints.  E.g.
```suggestion
    def __init__(self, config: Dict) -> None:
    """Stub constructor for ocean recentering task
    Parameters:
    ------------
    config: Dict
        configuration of XYZ
    Returns
    --------
    None
```
Please see examples of tasks in `ush/python/pygfs/task` of the global-workflow.